test/hypothesis_utils.py

```diff
@@ -304,11 +305,18 @@ def tensor_conv(
 
     return X, W, b, groups
 
-from hypothesis import settings
-settings.register_profile("no_deadline", deadline=None)
-settings.load_profile("no_deadline")
-
-# This is really just to get flake8 to not complain when this file
-# is imported purely for the side-effectful stuff above
+# We set the deadline in the currently loaded profile.
+# Creating (and loading) a separate profile overrides any settings the user
+# already specified.
+settings._profiles[settings._current_profile].__dict__['deadline'] = None
+settings._profiles[settings._current_profile].__dict__['timeout'] = hypothesis.unlimited
```

Comment:
is `hypothesis.unlimited` valid for hypothesis version < 3.27.0?

Reply:
Good point!